yoti_python_sdk/dynamic_sharing_service/policy/wanted_attribute_builder.py

```diff
@@ -8,7 +8,7 @@ class WantedAttributeBuilder(object):
     """
 
     def __init__(self):
-        self.__attribute = {}
+        self.__attribute = {"name": None}
```

Comment:
I think it's better that we don't default name to None here, but when we try to get it in the build method, set the default value to `None` if the key doesn't exist (see other comment)

```suggestion
        self.__attribute = {}
```